test/unit/ptable_test.rb

```diff
@@ -49,6 +52,7 @@ class PtableTest < ActiveSupport::TestCase
 
         pt = Ptable.find_by_name(name)
         assert_equal @os_deb.family, pt.os_family
+        assert_equal @new_org.id, pt.organization_ids
```

Comment:
This test is failing for me, `pt.organization_ids` is `[]` - moreover, you probably want `[@new_org.id]` as the first argument, else you're comparing Integer to Array.